mappings/net/minecraft/block/FlowerBlock.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_zhojkkij net/minecraft/block/FlowerBlock
+	FIELD f_bfitumpo SHAPE_OFFSET F
 	FIELD f_bhseogjj effectInStewDuration I
```

Comment:
Maybe stewEffectDuration

Reply:
Wasn't me that named that, though I could change it